src/Connections/GitlabIssue.ts

```diff
@@ -44,8 +44,8 @@ export class GitLabIssueConnection extends BaseConnection implements IConnection
     static readonly QueryRoomRegex = /#gitlab_(.+)_(.+)_(\d+):.*/;
     static readonly ServiceCategory = "gitlab";
 
-    static getTopicString(authorName: string, state: string) {
```

Comment:
Huh, (how) did this work before? Do we even use this?

Reply:
Don't think this was ever used :disappointed:

Reply:
I guess it's time to drop it then :)